content/desktop/contributing-and-collaborating-using-github-desktop/managing-tags.md

```diff
@@ -34,3 +34,15 @@ versions:
 
 3. All tags associated with the commit are visible in that commit's metadata.
 ![Viewing a tag in the commit](/assets/images/help/desktop/viewing-tags-in-commit.png)
+
+### Deleting tags
+
+{% note %}
+
+**Note**: You can only delete tags associated with unpushed commits.
```

Comment:
It might be nice to avoid "unpushed". How about:

```suggestion
**Note**: You can only delete tags associated with commits that have not yet been pushed.
```

Reply:
I think this is much more clear. Thanks!